src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

```diff
@@ -262,6 +262,8 @@ default ChangeInfo info() throws RestApiException {
     return get(EnumSet.noneOf(ListChangesOption.class));
   }
 
+  ChangeInfo getDetail() throws RestApiException;
```

Comment:
I cannot find this method in the official API: https://gerrit.googlesource.com/gerrit/+/master/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

The files in com.google.gerrit namespaces are (almost) plain copies from the official Gerrit source. Can you find this method with another name in the linked interface? Is it probably `ChangeApi#get(...)`?

Reply:
Found at https://gerrit.googlesource.com/gerrit/+/master/java/com/google/gerrit/server/restapi/change/ChangeRestApiModule.java#87

But not in the interface you linked.

It looks like the method is a shortcut for passing GetChange options.

Reply:
Are you able to fulfill you use case with the #get method? In that case, I'd suggest to close this PR without merging, because I'd like to keep the interface as near as possible to the official Gerrit one.

Reply:
I guess I would.
Let's close it for now. I will reopen if I can't do without it.
Thanks !